extensions/wikia/TemplateClassification/TemplateClassification.hooks.php

```diff
@@ -107,6 +108,13 @@ public static function onEditPageShowEditFormFields( \EditPage $editPage, \Outpu
 			$templateType = \TemplateClassificationService::NOT_AVAILABLE;
 		}
 
+		// Fallback to infobox on template draft conversion for not existent classification
```

Comment:
You can actually move it before the request above. If it is a Draft and the conversion param is there it means that it is an infobox.